rts/Sim/MoveTypes/GroundMoveType.cpp

```diff
@@ -638,7 +635,7 @@ void CGroundMoveType::StartMoving(float3 moveGoalPos, float moveGoalRadius) {
 	// units passing intermediate waypoints will TYPICALLY not cause any
 	// script->{Start,Stop}Moving calls now (even when turnInPlace=true)
 	// unless they come to a full stop first
-	ReRequestPath(true);
+	ReRequestPath(false);
```

Comment:
Hokomoko makes a good point about this change. This is called from Update, not SlowUpdate, so for now the force update should be set to true again.